src/Engine/ProtoAssociative/CodeGen.cs

```diff
@@ -3330,7 +3343,15 @@ private void EmitCallingForBaseConstructor(int thisClassIndex, ProtoCore.AST.Ass
                 if (bidx != Constants.kInvalidIndex)
                 {
                     baseConstructorName = core.ClassTable.ClassNodes[bidx].Name;
-                    int cidx = core.ClassTable.ClassNodes[bidx].ProcTable.IndexOf(baseConstructorName, argTypeList);
+                    int cidx = core.ClassTable.ClassNodes[bidx].ProcTable.GetFunctionBySignature(new ProcedureMatchOptions()
```

Comment:
Is `cidx` the class index of function index?

Reply:
same question - in the above calls to this function as well.